### PR TITLE
Merge relayer chain configs

### PR DESCRIPTION
**Pull request by mpetrun5**
Merged relayer and chain configs into single ChainBridge config file

## Description

- removed custom evm client config
- renamed SharedEVMConfig and SharedSubstrateConfig to EVMConfig and SubstrateConfig
- allowed app to dynamically add chains 
- added SetupDefaultEVMChain 

## Related Issue Or Context

Closes: #161 

## How Has This Been Tested? Testing details.

- unit tests
- e2e tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: e2e/evm-evm/example/cfg/config_evm-evm.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "relayer": {},
+  "chains": [
+    {
+      "id": 1,
+      "name": "evm1",
+      "type": "evm",
+      "endpoint": "ws://evm1-1:8546",
+      "from": "0xff93B45308FD417dF303D6515aB04D9e89a750Ca",
```
Comment (P1sar) [inline e2e/evm-evm/example/cfg/config_evm-evm.json:9]: looks like from is not used actually, lets remove it from example

**Review by P1sar — CHANGES_REQUESTED** [review]
Also could you please make corresponding changes to celo-module and example app in cooperation with @nmlinaric

File: e2e/evm-evm/example/cfg/config_evm-evm.json (hunk 1 above)
Comment (mpetrun5) [reply e2e/evm-evm/example/cfg/config_evm-evm.json:9]: from is used to pick private key from keystore

Comment (mpetrun5) [reply e2e/evm-evm/example/cfg/config_evm-evm.json:9]: oh sorry, misread
Will remove it from example.

File: flags/flags.go [hunk 2]
```diff
@@ -7,21 +7,17 @@ import (
...
 var (
 	// Flags for running the Chainbridge app
-	ChainConfigFlagName   = "config"
-	RelayerConfigFlagName = "relayer-config"
-	KeystoreFlagName      = "keystore"
-	BlockstoreFlagName    = "blockstore"
-	FreshStartFlagName    = "fresh"
-	LatestBlockFlagName   = "latest"
-	TestKeyFlagName       = "testkey"
+	ConfigFlagName      = "config"
```
Comment (P1sar) [inline flags/flags.go:10]: How do you think about renaming variable and value to something like ConfigPathFlagName = "cfgPath" or something like that. Just found it unclear

Comment (mpetrun5) [reply flags/flags.go:10]: Definitely, we can rename Flag to ConfigPathFlagName to keep it more simple for us, but I checked some CLIs and --config is pretty standard (wget, curl, docker).

**Review by MakMuftic — APPROVED** [review]

**Review by P1sar — APPROVED** [review]

**Pull request merged**